lib/fstab

```diff
@@ -1,3 +1,3 @@
 debugfs		/sys/kernel/debug	debugfs		defaults	0	0
 configfgs	/sys/kernel/config	configfs	defaults	0	0
-host		/host			9p		trans=virtio	0	0
+host		/host			9p		trans=virtio,ro	0	0
```

Comment:
This is definitely a good thing, but I think it's going to break too much stuff - leave this patch out of the pull request for now.

We'll need some sort of escape hatch at least; there are still places where tests need to e.g. build packages on the host filesystem from inside the VM.

Reply:
I figured this was probably overkill. Plus, I noticed that running Lustre tests over 9p can be painfully slow on some systems. `llmount.ktest` would take something like 20 minutes to run vs sub-1 minute on my local machine. Seems to be an issue with older QEMU, [based on this blog](https://linus.schreibt.jetzt/posts/qemu-9p-performance.html). I'll probably have to go back to rsync'ing Lustre/ZFS to the VM filesystem.

Reply:
I think we might be getting away from 9p in the future, I've heard something about a more direct path coming down the pipeline.

20 minutes? Ouch, is it a big copy to set up a test filesystem?

Reply:
Running `llmount.sh` shouldn't be copying that much from the root filesystem - just seems like older QEMU is just that inefficient. 9p seems to work perfectly fine on Debian 12.